packages/wast-parser/src/tokenizer.js

```diff
@@ -126,26 +126,32 @@ function tokenize(input: string) {
    * Can be used to look at the next character(s).
    *
    * The default behavior `lookahead()` simply returns the next character without consuming it.
+   * Letters are always returned in lowercase.
    *
    * @param int length How many characters to query. Default = 1
    * @param int offset How many characters to skip forward from current one. Default = 1
    *
    */
   function lookahead(length = 1, offset = 1) {
-    return input.substring(current + offset, current + offset + length);
+    return input
+      .substring(current + offset, current + offset + length)
+      .toLowerCase();
```

Comment:
I'm wondering if this could slow down the lookup process?

Reply:
Probably. The reason I moved it here is because every comparison of a lookahead with a single character (`lookahead() === `X`) had to deal with it separately which is much more error prone (easy to forget).

Reply:
Ok, I get that. We can still change it later.